### PR TITLE
Update README.rst and website sample code with `import pytest`

### PR DESCRIPTION
**Pull request by ltfschoen**
Thanks for submitting a PR, your contribution is really appreciated!

Here's a quick checklist that should be present in PRs:
- [ master] Target: for bug or doc fixes, target `master`; for new features, target `features`;

Unless your change is trivial documentation fix (e.g.,  a typo or reword of a small section) please:
- [ ] Make sure to include one or more tests for your change;
- [ ] Add yourself to `AUTHORS`;
- [ ] Add a new entry to `CHANGELOG.rst`
  - Choose any open position to avoid merge conflicts with other PRs.
  - Add a link to the issue you are fixing (if any) using RST syntax.
  - The pytest team likes to have people to acknowledged in the `CHANGELOG`, so please add a thank note to yourself ("Thanks @user for the PR") and a link to your GitHub profile. It may sound weird thanking yourself, but otherwise a maintainer would have to do it manually before or after merging instead of just using GitHub's merge button. This makes it easier on the maintainers to merge PRs. 

I am a new user to both Python and pytest. I wrote a basic Python program and refactored functions into a separate file for testing. 
I was only able to execute the tests with `pytest` after I added `import pytest` to the top of the file.
Note importantly that the pytest website should be updated accordingly particularly for beginners, as I discovered this issue when I was following the "Our first test run" example in the [getting started section of the pytest website](http://doc.pytest.org/en/latest/getting-started.html). Another example further down that page in the section "Asserting that a certain exception is raised" actually included `import pytest` which prompted me to added it to my code and I was then able to start using pytest.

### CODE REVIEW

**Comment by The-Compiler** [comment]
huh - I don't see why the import would be needed there?

File: README.rst [hunk 1]
```diff
@@ -19,10 +19,12 @@
 The ``pytest`` framework makes it easy to write small tests, yet
 scales to support complex functional testing for applications and libraries.
 
-An example of a simple test:
+An example of a simple test after `installation <http://doc.pytest.org/en/latest/getting-started.html>`_:
 
 .. code-block:: python
 
+    import pytest
```
Comment (RonnyPfannschmidt) [inline README.rst]: this block looks like it will break the documentation testing/regen

**Comment by nicoddemus** [comment]
@ltfschoen thanks for taking the time to write this. As others have commented, you don't really need to import pytest in order to execute simple tests such as those on the `README`. Could you remove the import please? The installation link seems to be useful.

**Comment by ltfschoen** [comment]
I've closed this PR due to overwhelming majority suggesting it is not required.

**Pull request closed without merging**